README.rst

```diff
@@ -0,0 +1,254 @@
+   |                                         |
+   |    $ wget http://opendata.cern.ch/...   |
+   +-----------------------------------------+
+               |
+               | ALIESD.root
+               V
+   +-----------------------------------------+
+   | (1) perform LEGO train test run         |
+   |                                         |
+   |    $ root -b -q './runEx01.C'           |
+   +-----------------------------------------+
+               |
+               | run.logs
+               | AnalysisResults.root
+               |
+               V
+   +----------------------------------------+
+   | (2) plot sample histogram              |
+   |                                        |
+   |    $ root -b -q plot.C                 |
+   +----------------------------------------+
+               |
+               | results/plot_pt.pdf
+               | results/plot_eta.pdf
+               |
+               V
+              STOP
+
+We shall see below how this sequence of commands is represented for the REANA
+serial workflow engine.
+
+
+5. Output results -
+-----------------------------
+
+
+The test run will create `ROOT <https://root.cern.ch/>`_ output files that
+usually contain histograms.
+
+.. code-block:: console
+
+   $ ls -l AnalysisResults.root
+   -rw-r--r-- 1 root root  31187 July 18 17:35 AnalysisResults.root
+
+The user typically uses the output files to produce final plots. For example,
+running ``plot.C`` output macro on the ``AnalysisResults.root`` output file will
+permit to visualise the pt distribution of the accepted events:
+
+.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-alice-pt-analysis/blob/master/docs/plot_pt.pdf
+   :alt: plot_pt.pdf
+   :align: center
+
+.. figure:: https://raw.githubusercontent.com/reanahub/reana-demo-alice-pt-analysis/blob/master/docs/plot_eta.pdf
+   :alt: plot_eta.pdf
+   :align: center
+
+
+
+Running the example on REANA cloud
+==================================
+
+We start by creating a `reana.yaml <reana.yaml>`_ file describing the above
+analysis structure with its inputs, code, runtime environment, computational
+workflow steps and expected outputs:
+
+.. code-block:: yaml
+
+    version: 0.3.0
+    inputs:
+      files:
+        - fix-env.sh
+        - data.txt
+        - AliAnalysisTaskEx01.cxx
+        - AliAnalysisTaskEx01.h
+        - runEx01.C
+        - plot.C
+        - OCDB.root
+      parameters:
+        data_location: http://opendata.cern.ch/record/1102/files/assets/alice/2010/LHC10h/000139038/ESD/0001/AliESDs.root
+    workflow:
+      type: serial
+      specification:
+        steps:
+          - environment: 'reanahub/reana-env-aliphysics:vAN-20180614-1'
+            commands:
+            - mkdir data/ && wget ${data_location} && mv AliESDs.root data/
+            - source fix-env.sh && root -b -q './runEx01.C' | tee run.log
+            - mkdir results && source fix-env.sh && root -b -q './plot.C'
```

Comment:
Since everything lives in one directory as per ALICE techniques, let's also create results there directly, without `results` directory. As we do for the LEGO train example.